soco/events.py

```diff
@@ -576,6 +575,13 @@ def time_left(self):
             time_left = self.timeout-(time.time()-self._timestamp)
             return time_left if time_left > 0 else 0
 
+# if you want to use a different port than 1400, set
+# soco.events.event_listener_port accordingly after importing but before
+# subscribing to an event
+
+# pylint: disable=C0103
+event_listener_port = 1400
```

Comment:
If you are going to have a global, you should put something in the soco.config module, rather than here. I agree with @poirier however. It is better just to add an extra parameter to the `start` method, with a default of 1400.